lib/scrabble.rb

```diff
@@ -1,7 +1,13 @@
 class Scrabble
 
   def score(word)
-    1
+    return 0 if word == nil
+    word_upcase = word.upcase.chars
+    sum = 0
+      word_upcase.each do |letter|
+        sum += point_values[letter]
+      end
+    return sum
```

Comment:
Like the way you've broken down this problem. Could benefit from taking each piece and moving it to a smaller method.